### DISCUSSION TITLE
Upgrade noobaa

### DISCUSSION THREAD

**Discussion by dkoci** [noobaa/noobaa-core#7165] [General]
Hi.
I am running noobaa operator on OKD (installed with noobaa CLI). I would like to upgrade noobaa to latest version but I dont know how to do it properly.

**Comment by guymguym** [comment]
Hi @dkoci 

See https://github.com/noobaa/noobaa-operator/issues/171#issuecomment-847759757.

Basically, once you update the noobaa-operator image to the new version it will roll out the new version, however, there are more yamls deployed during `noobaa install` so this is not enough, you can dump the yamls for every version with:
```
❯ noobaa install yaml >noobaa-install.yaml
INFO[0000] Dump CRD yamls...
INFO[0000] Dump operator yamls...
INFO[0000] Dump system yamls...
INFO[0000] ✅ Done dumping installation yaml
```

**Reply by dkoci** [reply]
I installed noobaa operator v5.11.0 from scratch. Is there no mngmt console anymore? Mngmt console route prints only version.

![image](https://user-images.githubusercontent.com/5565897/213877605-6bdbfbd4-2453-48e0-838d-39f7272394c7.png)

**Reply by liranmauda** [reply]
Hi @dkoci 
There is no mgmt console anymore: 
https://github.com/noobaa/noobaa-core/pull/6948 and https://github.com/noobaa/noobaa-core/pull/7005

**Reply by dkoci** [reply]
Which is the latest version which still has mgmt console? CLI is too confusing.

**Reply by liranmauda** [reply]
@dkoci 
5.10 is the last one.
I wouldn't use the UI, as we deprecated it 🙂 
Can you explain what is your use of the UI?

**Reply by dkoci** [reply]
I really like the UI. It is very simple, user friendly...
I need to create connection to our S3 storage, which is in our datacenter. Then I need to create buckets.
I tried some time ago with CLI but it was so confusing. Resource names was different than in the UI. I still dont get it what is bucket store, bucket class etc... 🙂

**Reply by liranmauda** [reply]
Personally I am using `aws cli`

**Reply by dkoci** [reply]
Sorry I was talking about noobaa cli. I am not using aws cli.

**Reply by liranmauda** [reply]
yes, I know 🙂 
I am using `aws cli`, configuring the secret and the key:
```
$ aws configure
AWS Access Key ID [****************2Wzk]:
AWS Secret Access Key [****************NNBJ]:
Default region name [None]:
Default output format [None]:
```
Then use the s3 or s3api and perform the s3 op that I need (mb for make bucket)
`aws s3 --endpoint-url=<url> mb s3://<bucketname> --no-verify-ssl` 
 
you can get the s3 URL by running noobaa CLI (noobaa status) and see it under the `S3 Addresses`

**Reply by guymguym** [reply]
Thanks you @dkoci, we are sorry that we couldn't save the UI from the devops cli scipting claws... We loved using it as well, but there was not enough demand to keep investing in it.
